x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_overview/components/recent_logs.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useValues } from 'kea';
+
+import {
+  EuiPageContent,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageContentBody,
+  EuiTitle,
+} from '@elastic/eui';
+
+import { EuiButton } from '../../../../shared/react_router_helpers';
+
+import { ENGINE_API_LOGS_PATH, getEngineRoute } from '../../../routes';
+import { RECENT_API_EVENTS } from '../../api_logs/constants';
+import { VIEW_API_LOGS } from '../constants';
+
+import { EngineLogic } from '../../engine';
+
+export const RecentLogs: React.FC = () => {
+  const { engineName } = useValues(EngineLogic);
+  const engineRoute = getEngineRoute(engineName);
+
+  return (
+    <EuiPageContent>
+      <EuiPageContentHeader responsive={false}>
+        <EuiPageContentHeaderSection>
+          <EuiTitle size="xs">
+            <h2>{RECENT_API_EVENTS}</h2>
+          </EuiTitle>
+        </EuiPageContentHeaderSection>
+        <EuiPageContentHeaderSection>
+          <EuiButton to={engineRoute + ENGINE_API_LOGS_PATH} size="s">
```

Comment:
Gotcha, I get that confusion - it should hopefully not be overly magical since the import source is right there at the top of the file, but I definitely get it - the find & replace point is a good one.

The thing I was struggling with originally was what to name it that wouldn't look or read super weird while parsing through JSX - usage wise, my intention was "It's the exact EUI same component, but react routing is taken care of for you". Naming wise everything I came up with felt super awkward (`<EuiReactRouterButton>`? `<EuiRRButton>`?) but if we can come up with something that flows well I'd be down to change it.

Just throwing some options out there:

1.
```
<EuiButtonTo />
<EuiLinkTo />
<EuiPanelTo />
```
2.
```
<EuiRouterButton /> or <EuiRoutedButton />?
<EuiRouterLink />
<EuiRouterPanel />
```

Thoughts? Suggestions?

Reply:
They're named the following at the source, why not just keep them as is? EuiReactRouterLink, etc.
```
export {
  EuiReactRouterLink as EuiLink,
  EuiReactRouterButton as EuiButton,
  EuiReactRouterPanel as EuiPanel,
} from './eui_link';
```

Reply:
Haha because I hated how long and wordy those names were. Especially for how often we use links/buttons etc., I want something that's easy to read and doesn't add a ton of characters (which could trigger multilines, etc.).

Sounds like this is a naming thing at this point though, so let me put this to a vote w/ the rest of the frontend team!

Reply:
PR follow up to this conversation: https://github.com/elastic/kibana/pull/83718